client/keys/utils.go

```diff
@@ -171,3 +136,8 @@ func printPubKey(info keys.Info, bechKeyOut bechKeyOutFn) {
 
 	fmt.Println(ko.PubKey)
 }
+
+func isRunningOnServer() bool {
+	backends := keyring.AvailableBackends()
+	return len(backends) == 2 && backends[1] == keyring.BackendType("file")
```

Comment:
What does running on server mean here?

Reply:
It's a misnomer, I copied it from the original PR and am happy to rename it. It attempts to determine whether the test is running on a developer machine or in a CI container.